src/app/content/courses/introduction-to-assembly/program-example/en.mdx

```diff
@@ -69,14 +69,15 @@ And this is the second instruction: `128 95 00 00 00 00 00 00 00`
 If we were to disassemble the binary to turn it back into compilable sBPF Assembly, the code would look like this:
 
 ```sbpf
-.globl entrypoint:
-    mov64 r0, 0x00   ; r0 <- success
-    exit             ; finish, return r0
+.globl entrypoint
+entrypoint:
+    mov64 r0, 0x00   # r0 <- success
+    exit             # finish, return r0
```

Comment:
Checked with @deanmlittle, we should standardize on using `//` for sBPF comments.

```suggestion
    mov64 r0, 0x00   // r0 <- success
    exit             // finish, return r0
```